qiskit/synthesis/linear_phase/cx_cz_depth_lnn.py

```diff
@@ -245,7 +245,7 @@ def synth_cx_cz_depth_line_my(mat_x: np.ndarray, mat_z: np.ndarray) -> QuantumCi
     n = len(mat_x)
     mat_x = calc_inverse_matrix(mat_x)
 
-    cx_instructions_rows_m2nw, cx_instructions_rows_nw2id = _optimize_cx_circ_depth_5n_line(mat_x)
+    cx_instructions_rows_m2nw, cx_instructions_rows_nw2id = optimize_cx_circ_depth_5n_line(mat_x)
```

Comment:
Ah, I see the problem: we actually want to have a method that returns both lists. So maybe we can expose two different functions from the Rust crate (the one that returns a pair of lists and the one that returns the full circuit), what do you think?

Reply:
Done in 4902815e1